src/server/actions/study.actions.ts

```diff
@@ -255,6 +257,7 @@ export const rejectStudyProposalAction = orgAction(
                 .executeTakeFirstOrThrow()
         })
 
+        await audit('REJECTED', userId, 'STUDY', studyId)
```

Comment:
I would find it easier to follow if the `audit` table accepted an object:
```suggestion
        await audit({ eventType: 'REJECTED', userId,  recordType: 'STUDY', recordId: studyId })
```